src/TecanScanner/TecanScanner.php

```diff
@@ -55,4 +57,28 @@ public static function parseRawContent(string $rawContent): FluidXPlate
 
         return $plate;
     }
+
+    /**
+     * Checks if a string can be parsed into a FluidXPlate.
+     */
+    public static function isValidRawContent(string $rawContent): bool
```

Comment:
When would we need such a method? Do we sometimes have strings that could represent a FluidXPlate, but could also be something different? If not, I would prefer the explainability offered by the errors when parsing